src/org/smssecure/smssecure/notifications/MessageNotifier.java

```diff
@@ -200,7 +203,19 @@ private static void sendSingleThreadNotification(Context context,
                                            context.getString(R.string.MessageNotifier_mark_as_read),
                                            notificationState.getMarkAsReadIntent(context, masterSecret));
       builder.addAction(markAsReadAction);
-      builder.extend(new NotificationCompat.WearableExtender().addAction(markAsReadAction));
+
+      String[] replyChoices = context.getResources().getStringArray(R.array.wear_replies);
+
+      RemoteInput remoteInput = new RemoteInput.Builder(EXTRA_VOICE_REPLY).setChoices(replyChoices).build();
+
+      Action replyAction = new Action.Builder(R.drawable.ic_reply_icon,
```

Comment:
Looks like the build is failing because it can't find the "ic_reply_icon" resource. Could you fix it and update your commit?